.github/workflows/timeseries-scans.yaml

```diff
@@ -0,0 +1,415 @@
+#
+# Apache v2 license
+# Copyright (C) 2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+name: "Time Series Analytics Microservice SDLe Scans"
+run-name: "Time Series Analytics Microservice SDLe Scans workflow (by @${{ github.actor }} via ${{ github.event_name }})"
+on:
+  workflow_dispatch:
+    inputs:
+      target:
+        description: 'Which Scans to run'
+        type: choice
+        options:
+        - all-scans

```

Comment:
Thanks @sathyendranv for getting this done so neatly, just explore if this pipeline can be leverage as is with few config changes to run in for our sample app so we can promote the reusability, we can have common stuff going under .github/actions and in the individual workflows (sample app and here), we have that cloning and build step which is specific.